### PR TITLE
Expose custom nonce parameters on sign()

### PR DESCRIPTION
**Pull request by mb300sd**

### CODE REVIEW

File: coincurve/keys.py [hunk 1]
```diff
@@ -26,16 +26,21 @@ def __init__(self, secret=None, context=GLOBAL_CONTEXT):
...
+    def sign(self, message, hasher=sha256, custom_nonce=None):
         msg_hash = hasher(message) if hasher is not None else message
         if len(msg_hash) != 32:
             raise ValueError('Message hash must be 32 bytes long.')
 
         signature = ffi.new('secp256k1_ecdsa_signature *')
 
+        nonce_fn = ffi.NULL
+        nonce_data = ffi.NULL
+        if custom_nonce
```
Comment (ofek) [inline coincurve/keys.py]: missing colon

Comment (mb300sd) [inline coincurve/keys.py]: oops, fixed.

Comment (ofek) [inline coincurve/keys.py]: Did you commit the fix?

Comment (mb300sd) [inline coincurve/keys.py]: It's showing on my branch, PR not updating for some reason :/ going to close and open new one.

**Pull request closed without merging**